src/custom/components/Markdown/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactNode } from 'react'
+import ReactMarkdown from 'react-markdown/with-html'
+import useFetchFile from 'hooks/useFetchFile'
+import { Title, Content, PageWrapper } from 'components/Markdown/markdown.styled'
+
+interface MarkdownParams {
+  content: string
+  title: ReactNode
+}
+
+export default function Markdown({ content, title }: MarkdownParams) {
+  const { error, file } = useFetchFile(content)
+  return (
+    <PageWrapper>
+      <Title>{title}</Title>
+      <Content>
+        {file && (
+          <ReactMarkdown allowDangerousHtml linkTarget="_blank">
+            {error ? error : file}
+          </ReactMarkdown>
+        )}
```

Comment:
Can you have `file` when `error` is set?
Otherwise the `error` will never be displayed

Reply:
split